packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerForm/AddPrayerFormConnected.js

```diff
@@ -11,12 +13,57 @@ class AddPrayerFormConnected extends React.Component {
 
   render() {
     return (
-      <Query query={getUserProfile} fetchPolicy={'cache-only'}>
-        {({ data }) => (
-          <AddPrayerForm
-            avatarSource={get(data, 'currentUser.profile.photo', { uri: null })}
-            {...this.props}
-          />
+      <Query query={GET_USER_PROFILE} fetchPolicy={'cache-only'}>
+        {(queryObj) => (
```

Comment:
If all you're using here is `data`, why hide it behind `queryObj`?

Reply:
```suggestion
        {({ data }) => (
```

Reply:
yea...I don't like it either. It's being used another scope below so I didn't know of a better way to do it.